front/app/modules/commercial/content_builder/admin/components/ContentBuilderToolbox/index.tsx

```diff
@@ -4,23 +4,24 @@ import React from 'react';
 import { useEditor, Element } from '@craftjs/core';
 
 // Router
-import { withRouter, WithRouterProps } from 'utils/cl-router/withRouter';
+import { useParams } from 'react-router-dom';
 
 // intl
-import { injectIntl } from 'utils/cl-intl';
+import { FormattedMessage, injectIntl } from 'utils/cl-intl';
 import { InjectedIntlProps } from 'react-intl';
 
 // components
 import ToolboxItem from './ToolboxItem';
-import { Box } from '@citizenlab/cl2-component-library';
+import { Box, Title } from '@citizenlab/cl2-component-library';
 import Text from '../CraftComponents/Text';
-import TwoColumn from '../CraftComponents/TwoColumn';
+import { TwoColumn } from '../CraftComponents/TwoColumn';
```

Comment:
Why is this export different? Should match the other ones.

Reply:
Ah sorry - something left from when I was testing things to get the TwoColumn working. I'll revert this :+1: